aspnetcore/mvc/views/overview.md

```diff
@@ -1,228 +1,336 @@
+You can also use a relative path to specify views in different directories without the *.cshtml* extension. Inside the `HomeController`, you can return the *Index* view of your *Manage* views with a relative path:
 
-<!-- literal_block {"ids": [], "linenos": false, "xml:space": "preserve", "language": "html", "highlight_args": {"hl_lines": [1]}} -->
+```csharp
+return View("../Manage/Index");
+```
 
-```html
+Similarly, you can indicate the current controller-specific directory with the "./" prefix:
+
+```csharp
+return View("./About");
+```
+
+[Partial views](xref:mvc/views/partial) and [view components](xref:mvc/views/view-components) use similar (but not identical) discovery mechanisms.
+
+You can customize the default convention for how views are located within the app by using a custom [IViewLocationExpander](/aspnet/core/api/microsoft.aspnetcore.mvc.razor.iviewlocationexpander).
+
+View discovery relies on finding view files by file name. If the underlying file system is case sensitive, view names are probably case sensitive. For compatibility across operating systems, match case between controller and action names and associated view folders and file names. If you encounter an error that a view file can't be found while working with a case-sensitive file system, confirm that the casing matches between the requested view file and the actual view file name.
+
+Follow the best practice of organizing the file structure for your views to reflect the relationships among controllers, actions, and views for maintainability and clarity.
+
+## Passing data to views
+
+You can pass data to views using several approaches. The most robust approach is to specify a [model](xref:mvc/models/model-binding) type in the view. This model is commonly referred to as a *viewmodel*. You pass an instance of the viewmodel type to the view from the action.
+
+Using a viewmodel to pass data to a view allows the view to take advantage of *strong* type checking. *Strong typing* (or *strongly-typed*) means that every variable and constant has an explicitly defined type (for example, `string`, `int`, or `DateTime`). The validity of types used in a view is checked at compile time.
+
+Tooling, such as [Visual Studio](https://www.visualstudio.com/vs/) or [Visual Studio Code](https://code.visualstudio.com/), can also list members (properties of a model) while you're adding them to a view, which helps you write code faster with fewer errors. This feature is called [IntelliSense](/visualstudio/ide/using-intellisense) in Microsoft tools.
```

Comment:
"Tooling, such as Visual Studio or Visual Studio Code, can also list members (properties of a model) while you're adding them to a view, which helps you write code faster with fewer errors. This feature is called IntelliSense in Microsoft tools."

100% accurate, but either needs a tie in to how Intellisense works with strongly typed viewmodels (the previous paragraph) or else it is just a random fact about VS squeezed into an article about Views and sounds somewhat off topic.
How about...
"Visual Studio and Visual Studio Code list strongly-typed class members using a feature called IntelliSense. For example, when you want to see properties of a ViewModel, type the variable name for the ViewModel followed by a period (`.`). This helps you write code faster with fewer errors."

Also a screenshot of intellisense at work would be helpful here.